harvest/storage/csv_storage.py

```diff
@@ -23,16 +25,18 @@ def __init__(self, save_dir: str='data'):
         """
         self.save_dir = save_dir
 
-        self.storage_lock.acquire()
```

Comment:
removes these because it causes an issue since `super().store` tries to also acquire this lock.